remote-processor/Message.h

```diff
@@ -44,7 +44,8 @@ class REMOTE_PROCESSOR_EXPORT CMessage
     {
         ECommandRequest,
         ESuccessAnswer,
-        EFailureAnswer
+        EFailureAnswer,
+        EInvalid = uint8_t(~0),
```

Comment:
- For invalid value we usually hardcode some arbitrary value like `205` or `187` or `66`... Matter of style though.
- Why `~0`, it was `-1` before, isn't it ?